thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/DetectionJobResource.java

```diff
@@ -314,8 +313,8 @@ public Response generateAnomaliesInTrainingData(@PathParam("id") @NotNull long i
    * @param functionId: the functionId to fetch merged anomalies with holidays removed
    * @param startTime: start time in milliseconds of merged anomalies
    * @param endTime: end time of in milliseconds merged anomalies
-   * @param holidayStarts: holidayStarts in milliseconds as string, in format {start1,start2,...}
-   * @param holidayEnds: holidayEnds in milliseconds as string, in format {end1, end2,...}
+   * @param holidayStarts: holidayStarts in ISO Format ex: 2016-5-23T00:00:00Z, start1,start2,...
```

Comment:
I think it is better to say, a list of ISO DateTime String (e.g. 2016-5-23T00:00:00Z) separated by comma.